shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -123,8 +126,7 @@ - (void)dealloc {
   [_children removeAllObjects];
   [_children release];
   _parent = nil;
-  [_container release];
-  _container = nil;
+  _container.get().semanticsObject = nil;
```

Comment:
this really worked?

Reply:
I can't remember the details now, but this was part of breaking the circular references.

Reply:
I can't remember the details now, but this was part of breaking the circular references.